src/ReinforcementLearningZoo/src/algorithms/tabular/td_learner.jl

```diff
@@ -3,6 +3,18 @@ export TDLearner
 using LinearAlgebra: dot
 using Distributions: pdf
 
+"""
+    TDLearner(;approximator, γ=1.0, method, n=0)
+
+Use temporal-difference method to estimate state value or state-action value.
+
+# Fields
+- `approximator` can be either
```

Comment:
```suggestion
- `approximator` can be either a
```
To emphasize that what comes after is a list of types, not a list of accepted values (to the difference with `method` below).